keras_cv/models/backbones/efficientnet_v1/efficientnet_v1_backbone.py

```diff
@@ -0,0 +1,484 @@
+                width_coefficient=width_coefficient,
+                divisor=depth_divisor,
+            )
+            output_filters = round_filters(
+                filters=output_filters,
+                width_coefficient=width_coefficient,
+                divisor=depth_divisor,
+            )
+
+            repeats = round_repeats(
+                repeats=num_repeats,
+                depth_coefficient=depth_coefficient,
+            )
+            strides = stackwise_strides[i]
+            squeeze_and_excite_ratio = stackwise_squeeze_and_excite_ratios[i]
+
+            for j in range(repeats):
+                # The first block needs to take care of stride and filter size
+                # increase.
+                if j > 0:
+                    strides = 1
+                    input_filters = output_filters
+
+                if strides != 1:
+                    pyramid_level_inputs.append(x.node.layer.name)
+
+                # 97 is the start of the lowercase alphabet.
+                letter_identifier = chr(j + 97)
+                x = apply_efficientnet_block(
+                    inputs=x,
+                    filters_in=input_filters,
+                    filters_out=output_filters,
+                    kernel_size=stackwise_kernel_sizes[i],
+                    strides=strides,
+                    expand_ratio=stackwise_expansion_ratios[i],
+                    se_ratio=squeeze_and_excite_ratio,
+                    id_skip=True,
+                    activation=activation,
+                    drop_rate=drop_connect_rate * block_id / blocks,
+                    name="block{}{}_".format(i + 1, letter_identifier),
+                )
+                block_id += 1
+
+        # Build top
+        top_filters = round_filters(
+            filters=1280,
+            width_coefficient=width_coefficient,
+            divisor=depth_divisor,
+        )
+
+        x = layers.Conv2D(
+            filters=top_filters,
+            kernel_size=1,
+            padding="same",
+            strides=1,
+            kernel_initializer=conv_kernel_initializer(),
+            use_bias=False,
+            name="top_conv",
+        )(x)
+        x = layers.BatchNormalization(
+            axis=3,
+            name="top_bn",
+        )(x)
+        x = layers.Activation(activation=activation, name="top_activation")(x)
+
+        pyramid_level_inputs.append(x.node.layer.name)
+
+        # Create model.
+        super().__init__(inputs=img_input, outputs=x, **kwargs)
+
+        self.include_rescaling = include_rescaling
+        self.width_coefficient = width_coefficient
+        self.depth_coefficient = depth_coefficient
+        self.dropout_rate = dropout_rate
+        self.drop_connect_rate = drop_connect_rate
+        self.depth_divisor = depth_divisor
+        self.activation = activation
+        self.input_tensor = input_tensor
+        self.pyramid_level_inputs = {
+            i + 1: name for i, name in enumerate(pyramid_level_inputs)
```

Comment:
We're now using strings for keys. See EfficientNetV1 ([link](https://github.com/keras-team/keras-cv/blob/deefe948bb2f9d72695b404b9ee1caa35ee16482/keras_cv/models/backbones/efficientnet_v2/efficientnet_v2_backbone.py#L260))